templates/index.js

```diff
@@ -54,6 +58,7 @@ module.exports = function () {
       'password_reset_required',
       'post_verify',
       'recovery',
+      'sms.1',
```

Comment:
Not advocating for or against, but why use template name as number? Others have an explicit name that make it a little easier to know what that template does.